Client/Frontend/Onboarding/ViewControllers/OnboardingDefaultSettingsViewController.swift

```diff
@@ -0,0 +1,255 @@
+            contentContainerView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor),
+            contentContainerView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            contentContainerView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+
+            contentStackView.topAnchor.constraint(greaterThanOrEqualTo: contentContainerView.topAnchor, constant: topPadding),
+            contentStackView.leadingAnchor.constraint(equalTo: contentContainerView.leadingAnchor, constant: leadingPadding),
+            contentStackView.trailingAnchor.constraint(equalTo: contentContainerView.trailingAnchor, constant: -trailingPadding),
+            contentStackView.bottomAnchor.constraint(greaterThanOrEqualTo: contentContainerView.bottomAnchor, constant: -bottomPadding),
+            textStackView.leadingAnchor.constraint(equalTo: contentStackView.leadingAnchor),
+            primaryButton.leadingAnchor.constraint(equalTo: contentStackView.leadingAnchor),
+            primaryButton.trailingAnchor.constraint(equalTo: contentStackView.trailingAnchor),
+            contentViewHeightConstraint
+        ])
+    }
+
+    private func updateLayout() {
+        titleLabel.text = viewModel.title
+        primaryButton.setTitle(viewModel.buttonTitle, for: .normal)
+    }
+
+    private func addViewsToView() {
+        createLabels(viewModel.descriptionSteps)
+
+        contentStackView.addArrangedSubview(titleLabel)
+        numeratedLabels.forEach { textStackView.addArrangedSubview($0)}
+        contentStackView.addArrangedSubview(textStackView)
+        contentStackView.addArrangedSubview(primaryButton)
+
+        containerView.addSubview(contentStackView)
+        contentContainerView.addSubview(containerView)
+        scrollView.addSubview(contentContainerView)
+        view.addSubview(scrollView)
+
+        view.backgroundColor = .white
+    }
+
+    private func createLabels(_ descriptionTexts: [String]) {
+        numeratedLabels = []
+        var attributedTexts: [NSAttributedString] = []
+        descriptionTexts.forEach { text in
+            let attributedText = MarkupAttributeUtility(baseFont: DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline, size: UX.numeratedTextFontSize)).addAttributesTo(text: text)
+            attributedTexts.append(attributedText)
+        }
+        attributedTexts.forEach { attributedText in
+            let label: UILabel = .build { label in
+                label.textAlignment = .left
+                label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .subheadline, size: UX.numeratedTextFontSize)
+                label.adjustsFontForContentSizeCategory = true
+//                label.accessibilityIdentifier = "\(self.viewModel.infoModel.a11yIdRoot)NumeratedLabel"
+                label.attributedText = attributedText
+                label.numberOfLines = 0
+            }
+            numeratedLabels.append(label)
+        }
```

Comment:
```suggestion
         let font = DynamicFontHelper.defaultHelper.preferredFont(
            withTextStyle: .subheadline,
            size: UX.numeratedTextFontSize)

        numeratedLabels = descriptionTexts.map { text in
            return UILabel = .build { label in
                label.font = font
                label.adjustsFontForContentSizeCategory = true
//                label.accessibilityIdentifier = "\(self.viewModel.infoModel.a11yIdRoot)NumeratedLabel"
                label.attributedText = MarkupAttributeUtility(baseFont: font).addAttributesTo(text: text)
                label.numberOfLines = 0
                label.textAlignment = .left
            }
        }
```
nit: a cleaner implementation that does everything in one step without unnecessary assignment of another array and a two loop iteration might look like this. (I wrote this in GitHub, so if you use this, please pull into Xcode and align/test it properly.) I would probably not even assign the font, and just pass that in directly, but that's just me. Assigning that does make things more legible, however.

Your solution is fine, I just wanted to explore a different way of doing this, so no need to incorporate if you don't want to. :)